### ISSUE TITLE
[Request] Audio only format; option for disable sample pictures

### ISSUE THREAD

**Issue by suliemail** [ram-on/SkyTube#118]
Hello!
It will be good if users can choose more formats for example audio only, video only and a combination of audio only, video only formats (like in youtube-dl).
The most important is audio only format, because a lot of people use youtube for playing music and they can save mobile data with this option. I think an option for disable sample pictures is good too.
Thank you ahead!

**Comment by avently** [comment]
Yeah, waiting this feature too. I want to limit my traffic when I use mobile data.

**Comment by OmlineEditor** [comment]
Add a video quality selection to the menu and there you can select only audio. I want to quickly select video quality for different videos


![68747470733a2f2f72616d2d6f6e2e6769746875622e696f2f536b79547562652f6173736574732f696d672f73637265656e73686f74732f706c61792e6a7067](https://user-images.githubusercontent.com/13348933/29532989-784f26d0-86b8-11e7-831d-d5bfacc60fe2.jpg)

**Comment by gzsombor** [comment]
This feature is getting closer to completion, I hope I can finish it for 3.0
